src/Sound.js

```diff
@@ -137,7 +137,7 @@ export default function Sound(props) {
           <div className="flex justfify-between py-3">
             {/* Download and edit buttons */}
             <div className="w-6/12 space-x-3">
-              <IonIcon size="large" icon={downloadSharp} />
+              {/* <IonIcon size="large" icon={downloadSharp} /> */}
```

Comment:
The download button works. This shouldn't be commented

Reply:
That's not the download button. That's the `<IonIcon size="large" icon={downloadSharp} />` Ionicon component that appears below the lefthand-side of a waveform.

Reply:
Ah right, makes sense. Thanks!

Reply:
You're welcome!